Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -61,8 +61,8 @@ public struct ChatMessage {
     ///
     /// If message is inline reply this property will contain the message quoted by this reply.
     ///
-    public var quotedMessage: ChatMessage? { _quotedMessage }
-    @CoreDataLazy internal var _quotedMessage: ChatMessage?
+    public var quotedMessage: ChatMessage? { _quotedMessage() }
+    let _quotedMessage: () -> ChatMessage?
```

Comment:
`public let quotedMessage: ChatMessage?` fails to compile due to the error of recursive reference of the type.